test/test_torch.py

```diff
@@ -6483,6 +6502,11 @@ def test_isinf_type(self, device):
         with self.assertRaises(TypeError):
             torch.isinf(1)  # Parameter must be a tensor
 
+    @onlyCPU
+    def test_isreal_type(self, device):
```

Comment:
I don't think you need this test. It doesn't test your function as much as the PyTorch framework, which should be tested elsewhere.